lib/objects/objectsInMemFileDB.js

```diff
@@ -0,0 +1,2301 @@
+/**
+ *      Object DB in memory - Server
+ *
+ *      Copyright 2013-2018 bluefox <dogafox@gmail.com>
+ *
+ *      MIT License
+ *
+ */
+
+/* jshint -W097 */
+/* jshint strict: false */
+/* jslint node: true */
+/* jshint -W061 */
+'use strict';
+
+const extend            = require('node.extend');
+const fs                = require('fs');
+const InMemoryFileDB    = require('../inMemFileDB');
+const tools             = require('../tools.js');
+const utils             = require('./objectsUtils');
+
+/** @class */
+class ObjectsInMemoryFileDB extends InMemoryFileDB {
```

Comment:
Add JSDoc comment describing what this adds.

Also I scrolled quickly thorugh this file as it is huge. So don't expect too much

Reply:
I did deeper check for the basic things you noticed